apps/sample-configs/osvr_server_config.moverio_highfov.json

```diff
@@ -0,0 +1,98 @@
+{
+  "description": "This configuration supports video (so-called 'positional') and IMU fusion tracking, in addition to orientation-only tracking, with the OSVR HDK 2, for units that were manufactured as HDK2 (NOT UPGRADED FROM HDK 1.x!). It is configured for RenderManager applications in direct mode on HDK2 optics.",
+  "display": {
+  "meta": {
+    "schemaVersion": 1
+  },
+  "hmd": {
+    "device": {
+      "vendor": "Epson",
+      "model": "Moverio BT-300 series",
+      "num_displays": 1,
+      "Version": "1.0",
+      "Note": "This is not the native FOV of the device but a higher FOV for 'magic window' style display."
```

Comment:
This is not how I understand the term magic window: as I understand it, is more like ARCore: viewing surface in the world that's tracked and has typically has a camera (video see-thru), not head-coupled.

Reply:
Imagine ARCore style magic window, but instead of holding it with your hand, you have your phone strapped to your head and held about half a foot in front of your face. That's what the Moverio looks like. If you have a better term for it, I'll use it.